parser/models.py

```diff
@@ -0,0 +1,151 @@
+from django.db import models

```

Comment:
Why we have two models files here?
Please remove one and update all the migrations.